oggm/tests/test_models.py

```diff
@@ -1377,6 +1377,96 @@ def test_inversion_parabolic(self):
             plt.plot(bed_shape_gl[:-3])
             plt.show()
 
+    def test_inversion_parabolic_sf_adhikari(self):
```

Comment:
Tests for each case would probably be much better with one single callable function and write wrappers, which are actually called for testing and set the shape factor mode before calling the function.